### ISSUE TITLE
Don't use Datastore emulator for tests

### ISSUE THREAD

**Issue by frankyn** [GoogleCloudPlatform/getting-started-ruby#69]
Remove references to gcd for emulating Datastore.

Also, update the tests to wait for Datastore operations to complete.  The live service is eventually consistent whereas the [emulator can be run with 100% consistency](https://cloud.google.com/sdk/gcloud/reference/beta/emulators/datastore/start#--consistency).

*Original question:*

> database.example.yml

```
$ gcd.sh create gcd-test-dataset-directory
$ gcd.sh start --testing ./gcd-test-dataset-directory/
```

> What is gcd.sh?

**Comment by remi** [comment]
The Google Cloud Datastore (gcd) emulator

What is the bug?

**Comment by remi** [comment]
gcd.sh usage: http://stackoverflow.com/questions/28055135/installing-google-cloud-datastore-gcd

Now, there is a [Datastore Emulator](https://cloud.google.com/datastore/docs/tools/datastore-emulator) that's a part of the gcloud SDK: `gcloud beta emulators datastore`

This code could be updated to use that.  I'd prefer that we deprecate the use of an emulator in our tests here, though.  While the emulator is useful to developers in general, I would rather our tests run against the real thing.

I think a good issue might be: *"Don't use Datastore emulator for tests"*

**Comment by frankyn** [comment]
It was more about providing information, because I just found it in the database.yml file. This was more of a reminder that I had come across it.

**Comment by jeffmendoza** [comment]
2-datastore and now step 7 use the live service. The container engine step still needs to be moved over. gcd can be removed from step 2 in https://github.com/GoogleCloudPlatform/getting-started-ruby/blob/master/spec/setup.sh#L49

Step 2 needs the `wait_until` test fixed introduced in #73 in `7-compute-engine/spec/features/book_management_spec.rb`
